packages/gateway/src/features/registration/schema.graphql

```diff
@@ -225,8 +225,11 @@ type History {
   user: User
   date: Date
   action: RegStatus
+  requester: String
+  hasShowedVerifiedDocument: Boolean
   statusReason: StatusReason
   reason: String
+  otherReason: String
```

Comment:
@yeasinhossain-dsi I wonder if you could add some documentation about where in `Task` are we storing these values?

Reply:
@Zangetsu101 I think we will tackle this documentation in https://github.com/opencrvs/opencrvs-core/issues/4060